packages/gateway/src/features/user/usersAPI.ts

```diff
@@ -39,11 +34,11 @@ export class UsersAPI extends RESTDataSource {
     }
 
     try {
-      const response = this.post('getUser', { email })
+      const response = await this.post('getUser', { body: { email } })
```

Comment:
You probably don't want to `await` here as that would just break the caching functionality. We want to cache the **Promise** in this case and not the **Resolved** value so that when multiple requests are made, we can just return the cached **Promise** that we created on the first request

Reply:
Same goes for the other requests in this file

Reply:
Cool yea, I was wondering about that.. Makes sense!